suite/runsuite_wrapper.pl

```diff
@@ -321,6 +321,7 @@
                                    'code_api|tool.drcachesim.threads' => 1, # i#4928
                                    'code_api,tracedump_text,tracedump_origins,syntax_intel|common.loglevel' => 1, # i#1807
                                    'code_api|tool.drcachesim.threads-with-config-file' => 1, # i#4954
+                                   'code_api|tool.drcachesim.coherence' => 1, # i#2417
```

Comment:
I don't see this test mentioned in #2417.  If you could look at the reason it's failing and paste that output into the issue (timeout, output mismatch, assert, crash).  Or is this #4673, or #4167, or #3971?

Reply:
In general please reply to each review request ("Done", or somesuch) and mark as Resolve if resolved.  It looks like this was ignored -- if there's a disagreement please reply and we can discuss.

Reply:
Sorry about that, thought I had replied. 

It appears to be hitting the timeout repeatedly.

Reply:
Please add that information to #2417

Reply:
added